_ml-commons-plugin/connectors.md

```diff
@@ -91,45 +90,31 @@ When enabled, the `backend_roles`, `add_all_backend_roles`, or `access_model` op
+### Node settings
 
-2. An **internal connector**, saved in the model index, can only be used with one remote model. Unlike a standalone connector, users only need access to the model itself to access an internal connector because the connection is established inside the model.
+Standalone connectors consume less resources. Therefore, you can deploy any model from a standalone connector using data nodes. To make sure that your standalone connection uses data nodes, set `plugins.ml_commons.only_run_on_ml_node` to `false`, as shown in the following example:
 
-## Configuration options
+```json
+PUT /_cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.only_run_on_ml_node": false
+    }
+}
 
-The following configuration options are **required** in order to create a connector. These settings can be used for both standalone and internal connectors.
+```
+{% include copy-curl.html %}
 
-| Field | Data type | Description |
-| :---  | :--- | :--- |
-| `name` | String | The name of the connector. |
-| `description` | String | A description of the connector. |
-| `version` | Integer | The version of the connector. |
-| `protocol` | String | The protocol for the connection. For AWS services such as Amazon SageMaker and Amazon Bedrock, use `aws_sigv4`. For all other services, use `http`. |
-| `parameter` | JSON array | The default connector parameters, including `endpoint` and `model`. 
-| `credential` | String | Defines any credential variables required to connect to your chosen endpoint. ML Commons uses **AES/GCM/NoPadding** symmetric encryption with a key length of 32 bytes. When a connection cluster first starts, the key persists in OpenSearch. Therefore, you do not need to manually encrypt the key.
-| `action` | JSON array | Tells the connector what actions to run after a connection to ML Commons has been established.
-| `backend_roles` | String | A list of OpenSearch backend roles. For more information about setting up backend roles, see [Assigning backend roles to users]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control#assigning-backend-roles-to-users).
-| `access_mode` | String | Sets the access mode for the model, either `public`, `restricted`, or `private`. Default is `private`. For more information about `access_mode`, see [Model groups]({{site.url}}{{site.baseurl}}/ml-commons-plugin/model-access-control#model-groups).
-| `add_all_backend_roles` | Boolean | When set to `true`, adds all `backend_roles` to the access list, which only a user with admin permissions can adjust. When set to `false`, non-admins can add `backend_roles`.
+If you want to use an internal connector, use a dedicated ML node and set `plugins.ml_commons.only_run_on_ml_node` to `true`. Remember, using dedicated ML nodes consumes more resources, which could lead to increased costs when deploying models from an internal connector.
```

Comment:
> If you want to use an internal connector, use a dedicated ML node and set `plugins.ml_commons.only_run_on_ml_node` to `true`.

Not correct for this sentence. For remote model with internal connector, it also consumes less resources, no need to run on dedicated ML node.